### PR TITLE
Serve extension scripts and stylesheets as URLs

### PR DESCRIPTION
**Pull request by spadgett**
Switch extension scripts and stylesheets to use a URL rather than serving them from the master filesystem. The console index.html will reference the URLs directly in its `<script>` and `<link>` tags.

We plan to change the asset config extension scripts and stylesheets property names to make it clear that they're URLs as a follow-on.

/assign @jwforres 
@deads2k @sspeiche FYI

### CODE REVIEW

File: pkg/assets/apiserver/asset_apiserver.go [hunk 1]
```diff
@@ -129,16 +129,6 @@ func (c completedAssetServerConfig) New(delegationTarget genericapiserver.Delega
...
 		return nil, err
 	}
-	if err := c.addExtensionScripts(s.GenericAPIServer.Handler.NonGoRestfulMux); err != nil {
-		return nil, err
-	}
-	if err := c.addExtensionStyleSheets(s.GenericAPIServer.Handler.NonGoRestfulMux); err != nil {
-		return nil, err
-	}
-	c.addExtensionFiles(s.GenericAPIServer.Handler.NonGoRestfulMux)
-	if err := c.addWebConsoleConfig(s.GenericAPIServer.Handler.NonGoRestfulMux); err != nil {
```
Comment (jwforres) [inline pkg/assets/apiserver/asset_apiserver.go]: surprised this part is removed from looking at it

**Comment by spadgett** [comment]
@stevekuznetsov I assume this is a flake?

https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin_web_console_server_check/11/console

```
########## STARTING STAGE: RECORD THE STARTING METADATA ##########
+ [[ -s /var/lib/jenkins/jobs/test_pull_request_origin_web_console_server_check/workspace/activate ]]
+ source /var/lib/jenkins/jobs/test_pull_request_origin_web_console_server_check/workspace/activate
++ export VIRTUAL_ENV=/var/lib/jenkins/origin-ci-tool/352a8218d81bdad0819720bdd72fc1fd04c1c2de
++ VIRTUAL_ENV=/var/lib/jenkins/origin-ci-tool/352a8218d81bdad0819720bdd72fc1fd04c1c2de
++ export PATH=/var/lib/jenkins/origin-ci-tool/352a8218d81bdad0819720bdd72fc1fd04c1c2de/bin:/sbin:/usr/sbin:/bin:/usr/bin
++ PATH=/var/lib/jenkins/origin-ci-tool/352a8218d81bdad0819720bdd72fc1fd04c1c2de/bin:/sbin:/usr/sbin:/bin:/usr/bin
++ unset PYTHON_HOME
++ export OCT_CONFIG_HOME=/var/lib/jenkins/jobs/test_pull_request_origin_web_console_server_check/workspace/.config
++ OCT_CONFIG_HOME=/var/lib/jenkins/jobs/test_pull_request_origin_web_console_server_check/workspace/.config
++ mktemp
+ script=/tmp/tmp.aYuVZ0Krin
+ cat
+ chmod +x /tmp/tmp.aYuVZ0Krin
+ scp -F ./.config/origin-ci-tool/inventory/.ssh_config /tmp/tmp.aYuVZ0Krin openshiftdevel:/tmp/tmp.aYuVZ0Krin
+ ssh -F ./.config/origin-ci-tool/inventory/.ssh_config -t openshiftdevel 'bash -l -c "timeout 300 /tmp/tmp.aYuVZ0Krin"'
+ cd /data/src/github.com/openshift/aos-cd-jobs
+ trap 'exit 0' EXIT
+ sjb/gcs/started.py
++ exit 0
++ export status=FAILURE
++ status=FAILURE
+ set +o xtrace
########## FINISHED STAGE: FAILURE: RECORD THE STARTING METADATA [00h 05m 00s] ##########
```

**Comment by stevekuznetsov** [comment]
Yeah that is a weird one.

**Comment by jwforres** [comment]
/lgtm
since we plan to update the names of the fields as a follow on

**Comment by spadgett** [comment]
@abhgupta @jupierce Starting in 3.9, the web console will expect extension stylesheets and extension scripts to be URLs rather than files on the master filesystem. I'm working on openshift-ansible changes in https://github.com/openshift/openshift-ansible/pull/6359

**Pull request merged**